autoPyTorch/ensemble/ensemble_selection.py

```diff
@@ -231,13 +231,15 @@ def get_models_with_weights(
         """
         Handy function to tag the provided input models with a given weight.
 
-        Arguments:
+        Args:
             models (List[Tuple[float, BasePipeline]]):
                 A dictionary that maps a model's name to it's actual python object.
 
         Returns:
-            List[Tuple[float, BasePipeline]]:
-                each model with the related weight, sorted by performance
+            output (List[Tuple[float, BasePipeline]]):
+                each model with the related weight, sorted by ascending
```

Comment:
sorted by the ascending order with respect to the performance

Reply:
Ok, but errors in grammar are not good sometimes, because it can cause some misunderstandings. I addressed this point, so basically I could understand this statement (just a grammar issue).